src/features/imageUpload/imageLoader.js

```diff
@@ -0,0 +1,71 @@
+function isImageUri(uri) {
+	return /\.(gif|png|jpg|jpeg)$/i.test(uri);
+}
+
+function isFile(item) {
+	return item instanceof File || item instanceof Blob;
+}
+
+function isImageFile(item) {
+	// TODO: can probably change this to just be a regex: /image.*/
+	return item.type && (
+		_.includes(item.type, "/jpeg") ||
+		_.includes(item.type, "/jpg") ||
+		_.includes(item.type, "/gif") ||
+		_.includes(item.type, "/bmp") ||
+		_.includes(item.type, "/png"));
+}
+
+function getFileError(file) {
+	if (!isImageFile(file)) {
+		return "Unsupported file type.  Only images are supported.";
+	}
+	else if (file.size > 4 * 1024 * 1024) {
+		return "The file is too large! Files can be a maximum of 4MB.";
+	}
+
+	return null;
+}
+
+export default {
```

Comment:
I would personally prefer you `export const loadImage = () => ` vs this default. It matches the other functions better that way. I think that's how we've done this sort of thing other places.